modules/rosa-delete-users.adoc

```diff
@@ -0,0 +1,20 @@
+
+// Module included in the following assemblies:
+//
+// getting_started_rosa/rosa-creating-cluster.adoc
+
+
+[id="rosa-delete-users"]
+= Removing a user through the OCM console
+To remove full access of users so that they cannot access the cluster, you must remove the users through the OpenShift Cluster Manager (OCM) console in addition to revoking their `dedicated-admin` or `cluster-admin` privileges.
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
+* You have the IDP user name for the user whose privileges you are revoking.
+* You are logged in to the cluster with with `dedicated-admin` or `cluster-admin` privileges.
+
+.Procedure
+
+. From the *Administrator* perspective in the OpenShift web console, navigate to the *Cluster Details* page and click the *Access control* tab.
```

Comment:
Is it the OpenShift web console or OCM? I think it should OCM console here.

Reply:
Made the change.